contracts/ServiceNodeRewards.sol

```diff
@@ -75,6 +78,9 @@ contract ServiceNodeRewards is Initializable, Ownable2StepUpgradeable, PausableU
         liquidateTag = buildTag("BLS_SIG_TRYANDINCREMENT_LIQUIDATE");
         hashToG2Tag = buildTag("BLS_SIG_HASH_TO_FIELD_TAG");
         signatureExpiry = 10 minutes;
+        claimThreshold = 2_000_000 * 1e9;
+        periodicClaims = 0;
+        epochDay = 0;
```

Comment:
I think we want to change this to a 1M threshold, and using epoch "half days" instead of epoch days.